modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/timeline/CTCollectionHistoryProviderRegistry.java

```diff
@@ -74,7 +89,7 @@ protected void activate(BundleContext bundleContext) {
 
 					try {
 						emitter.emit(
-							_classNameLocalService.getClassNameId(
+							classNameLocalService.getClassNameId(
```

Comment:
@GabrielHenriquedev It's OK to directly use `ClassNameLocalServiceUtil` instead of using `Snapshot` to get the service in the static context.